app/views/security_group/_fw_rules.haml

```diff
@@ -0,0 +1,91 @@
+%h3
+  = _('Firewall Rules')
+  %table.table.table-bordered
+    %thead
+      %tr
+        %th
+          = _('Direction')
+        %th
+          = _('Network Protocol')
+        %th
+          = _('Host Protocol')
+        %th
+          = _('Remote Security Group (name - ref)')
+        %th
+          = _('IP Range')
+        %th
+          = _('Port')
+        %th
+          = _('End Port')
+        %th
+          = _('Action')
+    %tbody
+      %tr{"ng-repeat" => "fw_rule in vm.securityGroupModel.firewall_rules",
+          "ng-form"   => "rowForm",
+          "ng-if"     => "!fw_rule.deleted"}
+        %td.table-view-pf-select{"ng-class" => "{'has-error': angularForm.rowForm.direction.$error.required && vm.securityGroupModel.firewall_rules[$index].direction === ''}"}
+          %select{"name"                        => "direction",
+            "data-width"                  => "auto",
+            "ng-model"                    => "vm.securityGroupModel.firewall_rules[$index].direction",
+            "ng-options"                  => "direction for direction in vm.directions",
+            "ng-selected"                 => "fw_rule.direction",
+            "required"                    => "",
+            "selectpicker-for-select-tag" => ""}
+            %option{"value" => ""}
+              = "<#{_('Choose')}>"
+          %span.help-block{"ng-show" => "angularForm.rowForm.direction.$error.required && vm.securityGroupModel.firewall_rules[$index].direction === ''"}
+            = _("Required")
```

Comment:
I've simplified it, value is "inbound" by default, only other value possible is "outbound", this way no need to enforce require, there is no escape.